RELEASING.md

```diff
@@ -34,26 +34,14 @@ Release Steps
 
 ### Preparing Documentation
 
-Update version-sensitive files in the root and documentation sources
-in the docs/ subdirectory off the root.
+Use the [`docs/build-and-copy-docs.sh`](docs/build-and-copy-docs.sh) script to
+build a set of docs for this release and copy them to the cloned
+`docs.janusgraph.org` repo which you will update later.
 
-Documentation pages:
+You may also need to update the following files in the main repo for any new
+or updated dependencies:
 
-* changelog.txt
-* versions.txt
-* upgrade.txt
```

Comment:
I couldn't find instructions elsewhere that remind us to update the above files under `docs/`.  I think this is an important step to be documented. How do you plan to handle?

Reply:
Sorry, that was an extra deletion; I should have only deleted the next list of bulleted items (copying the same files to the top-level of the repo). Added back, cleaned up formatting.